plugins/woocommerce/includes/class-woocommerce.php

```diff
@@ -634,10 +634,6 @@ public function includes() {
 		/**
 		 * Tracks.
 		 */
-		include_once WC_ABSPATH . 'includes/tracks/class-wc-tracks.php';
-		include_once WC_ABSPATH . 'includes/tracks/class-wc-tracks-event.php';
-		include_once WC_ABSPATH . 'includes/tracks/class-wc-tracks-client.php';
-		include_once WC_ABSPATH . 'includes/tracks/class-wc-tracks-footer-pixel.php';
 		include_once WC_ABSPATH . 'includes/tracks/class-wc-site-tracking.php';
```

Comment:
This is needed for the actual init action

Reply:
so this is needed for the frontend context, is it?

Reply:
Updated by moving the logic around the action instead.